docker/internal-auth/compose.dedicated-plugins.yaml

```diff
@@ -20,6 +20,7 @@ services:
 
   teams-app:
     environment:
+      FEATURE_FLAG_ENABLE_INVITATIONS: false
```

Comment:
Hard code disable by setting `FEATURE_FLAG_ENABLE_INVITATIONS` to `false`, even if user sets to true in `.env`.